### DISCUSSION TITLE
FFmpeg VAAPI problem with OBS and AMD

### DISCUSSION THREAD

**Discussion by Botler85** [obsproject/obs-studio#11466] [General]
There is a problem with the encoder. I have set the resolution in OBS to 3440x1440, but the recorded file has a resolution of 3456x1440. Or 1920x1080 is 1920x1082. 

Does anyone know anything about this? Are there any encoder settings? Many thanks!

![Bildschirmfoto vom 2024-10-30 11-47-51](https://github.com/user-attachments/assets/b64c5102-6208-482d-8de2-58edd2f6fc03)
![Bildschirmfoto vom 2024-10-30 11-46-48](https://github.com/user-attachments/assets/4d53bc82-1378-4801-be9c-afa5d2e49cdd)
![Bildschirmfoto vom 2024-10-30 11-46-15](https://github.com/user-attachments/assets/f1e2ddb3-9e67-42d0-bcc0-c9ff9118ddf1)
![Bildschirmfoto vom 2024-10-30 11-46-02](https://github.com/user-attachments/assets/ee1c5b5b-331b-462a-9699-be7ece2a2177)
![Bildschirmfoto vom 2024-10-30 11-45-42](https://github.com/user-attachments/assets/66936820-eb99-41f2-b8b0-2c821ac965eb)
![Bildschirmfoto vom 2024-10-30 11-45-27](https://github.com/user-attachments/assets/cdc69b92-0be3-4360-af3f-16128be370ce)

**Comment by derrod** [comment]
It's a hardware limitation in AMD's first generation AV1 encoder: https://gitlab.freedesktop.org/mesa/mesa/-/issues/9185#note_1954937

It can only properly encode resolutions with a width divisible by 64 and a height divisible by 16, but they have a special hack for 1080p so it's "only" 1082p instead of 1088p.

**Reply by Botler85** [reply]
Sorry buddy, I don't get it :-)
For me that means I have to set the resolution differently in OBS. Maybe to 1920x1082?
